.appveyor.yml

```diff
@@ -0,0 +1,33 @@
+version: '{build}'
+skip_tags: true
+clone_depth: 10
+environment:
+  MAVEN_VERSION: 3.6.2
+  matrix:
+    - JAVA_HOME: C:\Program Files\Java\jdk1.8.0
+install:
+  - ps: |
+      Add-Type -AssemblyName System.IO.Compression.FileSystem
+      if (!(Test-Path -Path "C:\maven" )) {
+        Write-Host "Downloading Maven $env:MAVEN_VERSION"
+        (new-object System.Net.WebClient).DownloadFile("https://repo1.maven.org/maven2/org/apache/maven/apache-maven/$env:MAVEN_VERSION/apache-maven-$env:MAVEN_VERSION-bin.zip", 'C:\maven-bin.zip')
+        [System.IO.Compression.ZipFile]::ExtractToDirectory("C:\maven-bin.zip", "C:\maven")
+      }
+  - cmd: SET M2_HOME=C:\maven\apache-maven-%MAVEN_VERSION%
+  # Prepend Java entry, remove Ruby entry (C:\Ruby193\bin;) from PATH
+  - cmd: SET PATH=%M2_HOME%\bin;%JAVA_HOME%\bin;%PATH:C:\Ruby193\bin;=%;
+  - cmd: SET MAVEN_OPTS=-XX:MaxPermSize=1g -Xmx2g
+  - cmd: SET JAVA_OPTS=-XX:MaxPermSize=1g -Xmx2g
+  - cmd: mvn --version
+  - cmd: java -version
+build_script:
+  - mvn clean site -Psite --errors --batch-mode
+  - mvn clean install -Pqulice --errors --batch-mode
+
+cache:
+  - C:\maven\ -> appveyor.yml
+  - C:\Users\appveyor\.m2\ -> pom.xml
+
+notifications:
```

Comment:
@mohamednizar I think that we can suppress this, I think it's better to set them on Github Web INterface

Reply:
@paulodamaso Perfect, GitHub Support for Web interface notification,See the image given below.  I have remove the notifications parameters. 

![image](https://user-images.githubusercontent.com/8711284/66247560-7bf12a00-e73b-11e9-8af5-828e8434bebe.png)